### ISSUE TITLE
follow up: make port 19001 configurable

### ISSUE THREAD

**Issue by zirain** [envoyproxy/gateway#1703]
EG use 0.0.0.0:19001 for health check and prometheus metric, should make this configuration.

**Comment by arkodg** [comment]
+1 to making it configurable, that is something that can be done in the next release in v0.6, but shouldnt we have a dedicated port for prometheus, maybe `19002` by default ?

**Comment by zirain** [comment]
> +1 to making it configurable, that is something that can be done in the next release in v0.6, but shouldnt we have a dedicated port for prometheus, maybe `19002` by default ?

why not share the same port with health check?

**Comment by arkodg** [comment]
good point, if worker thread is different for different requests / starvation is unlikely, this is fine, until we add an API to make it configurable

**Comment by arkodg** [comment]
yah both issues make sense @shawnh2
